src/pages/partners/_premium_agent.module.scss

```diff
@@ -0,0 +1,51 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+
+  .wrapper {
+    width: 100%;
+    max-inline-size: 56rem;
+  }
+
+  .array_wrapper {
+    display: grid;
+    grid-row-gap: 2.5rem;
+    grid-column-gap: 5rem;
+    inline-size: 296px;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(5, 1fr);
+    margin-block-start: 2.5rem;
+
+    .country {
+      display: flex;
+      align-items: center;
+      gap: 1rem;
+    }
+
+    @media (min-width: 600px ) {
+      inline-size: 54rem;
+      gap: 4rem;
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: repeat(3, 1fr);
+    }
+  }
+
+  .heading{
+    max-inline-size: 27rem;
+
+    @include breakpoints(laptop) {
+      max-inline-size: 38rem;
+    }
+
+    .heading_countries{
+      max-inline-size: 32rem;
+
+      @include breakpoints(laptop) {
+        max-inline-size: 38rem;
+      }
+    }
+  }
+
+  .box{
+    max-inline-size: 56rem;
+  }
```

Comment:
Missing EOF 🙏

Reply:
No need for EOF bracket since every class has closing tag